### PR TITLE
wlroots: update to 0.13.0.

### PR DESCRIPTION
**Pull request by ifreund**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

This will remain a draft until all reverse dependencies have been bumped as needed.

### CODE REVIEW

**Comment by ericonr** [comment]
Probably makes sense to wait on #29641

**Comment by jbeich** [comment]
For wayfire/hikari/etc fixes try extracting from https://github.com/freebsd/freebsd-ports/commit/70148c11889b8de458a1c3ea23c94d05af404296

**Comment by ifreund** [comment]
@jbeich Thanks, I'm amazed by your speed as usual.

Looks like you've even patched my own river and zig-wlroots for wlroots 0.13.0 before I finished doing the same thing... If you give me a couple more hours next time it could save some duplicated effort :P Very cool to see river packaged for FreeBSD!

**Comment by ericonr** [comment]
Btw, I think we can remove all the `INSTALL.msg`s from wlroots compositors now.

**Comment by st3r4g** [comment]
for me wlroots 0.13 broke screen sharing on an Intel Broadwell GPU: it was really a xdg-desktop-portal-wlr bug (triggered by the wlroots update), now fixed in master, but it seems they haven't cut a point release yet.

**Comment by st3r4g** [comment]
could you add xdg-desktop-portal-wlr-0.3 here too? It has just been released

**Comment by ifreund** [comment]
@st3r4g Done, and I added your patch as that logic is obviously wrong and it works for me.

**Comment by st3r4g** [comment]
already merged upstream too.

File: srcpkgs/hikari/template [hunk 1]
```diff
@@ -25,13 +25,19 @@ pre_build() {
...
 	# bmake tends to ignore CC in the environment
 	make_build_args+=" CC=$CC"
 }
 
+# bmake's -q flag seems to differ in behavior from gnu make which causes the
+# build style's handling of the check target not existing to fail.
+do_check() {
+	:
+}
+
```
Comment (ahesford) [inline srcpkgs/hikari/template:40]: Use `make_check=no` to disable checks

Comment (ifreund) [reply srcpkgs/hikari/template]: done :+1:

**Comment by ericonr** [comment]
Updated the commit message for sway :)

**Pull request merged**